runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -58,6 +60,18 @@
 @Slf4j
 public class ItemManager
 {
+	private static final List<Pattern> normalizePatterns = new ArrayList<Pattern>()
```

Comment:
Is there any way we can run a test for this? Seems like we'd want to make sure this doesn't break with future updates.

Reply:
also, UPPER_CASE